src/pages/changelog/post-15.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "Improved Data and Resource Management"
+date: "16 Aug 2023"
+layout: "@changelogs"
+description: "You now have more control over your data and have the ability to delete any resource you create. You can also update environment variables from the CLI and have longer sessions."
+---
+
+import DetaHeart from '@/components/DetaHeart.astro'
+import Image from '@/components/Image.astro'
+
+## Longer Sessions
```

Comment:
Can we put this section at bottom?

Reply:
thought this is something a lot of users cared about and since it is short I put it at the top

Reply:
Also bundle it with `space env` so that all the delete stuff is together.

But we could also bring `space env` up here. I liked how you ended on delete account

Reply:
yeah I also had both at the end but liked the ending with delete account.